heft-plugins/heft-isolated-typescript-transpile-plugin/README.md

```diff
@@ -0,0 +1,12 @@
+# @rushstack/heft-isolated-typescript-transpile-plugin
+
+This is a Heft plugin for using swc as an isolated module transpiler
```

Comment:
I'm curious why this contains `typescript` in the plugin name instead of `swc`?

Reply:
We're currently treating swc as an implementation detail. As written, this plugin exists just to transpile TypeScript (without typechecking).